Sources/SwiftGenKit/Stencil/JSONParser+Context.swift

```diff
@@ -23,19 +23,12 @@ extension JSON.Parser {
   }
 
   private func map(file: JSON.File) -> [String: Any] {
-    let document = map(document: file.document)
-    return [
+    [
       "name": file.name,
       "path": file.path.string,
-      "document": document,
-      "documents": [document] // Deprecated: remains for legacy/compatibility reasons; will be removed in 7.0
-    ]
-  }
-
-  private func map(document: Any) -> [String: Any] {
-    [
-      "data": document,
-      "metadata": Metadata.generate(for: document)
+      "document": StencilContextLazyDocument(data: file.document),
+      // Deprecated: remains for legacy/compatibility reasons; will be removed in 7.0
+      "documents": [StencilContextLazyDocument(data: file.document)]
```

Comment:
Not sure if it's really a problem or not, but here we're creating 2 instances of the `StencilContextLazyDocument` class, once for `document` and one for `documents`. Wouldn't it be better to use the same instance in both instead?